### ISSUE TITLE
Fyne widget demo app not starting

### ISSUE THREAD

**Issue by PaulDotSH** [fyne-io/fyne#2544]
### Describe the bug:
You cannot open the fyne_demo app anymore

### To Reproduce:
Enter in the terminal
$ go get fyne.io/fyne/v2/cmd/fyne_demo/
Try to launch
$ fyne_demo

### Device (please complete the following information):
 - **OS:** Linux
 - **Version:** 5.14.7
 - **Go version:** 1.17.1
 - **Fyne version:** 2.1

**Comment by PaulIRL** [comment]
> That is most likely because the gobin path isn't added to your path. You need to add `~/go/bin` to your PATH variable. Without doing that, it can always be run with `~/go/bin/fyne_demo`.

Oh god my bad, that worked, one more question, is there the source code for the demo available somehere? Could you please link it to me?

**Comment by PaulIRL** [comment]
Another fix people might want for windows is installing tdm64 instead of other mingw binaries, tdm64 is the only software that made fyne compile on windows

**Comment by elibroftw** [comment]
Hey! I followed the instructions and at first thought I had done something wrong. The instructions foremost don't tell you to use '-d' when installing the demo and secondly, the instructions did not warn about the compilation time. I was waiting for almost a minute thinking something was wrong. Is it normal to take 1+ to compile?

**Comment by andydotxyz** [comment]
Please don’t add new issues to an existing, closed ticket. The documentation should be correct - please open a new issue with all the details requested if you have found a problem.

Go tools are changing subtly each version, we cannot help with just the information above.

**Comment by Jacalz** [comment]
> Hey! I followed the instructions and at first thought I had done something wrong. The instructions foremost don't tell you to use '-d' when installing the demo and secondly, the instructions did not warn about the compilation time. I was waiting for almost a minute thinking something was wrong. Is it normal to take 1+ to compile?

Do you mean that the `go get` mention should use the `-d` flag? The compile time is mentioned at the bottom of the page if you look in the installing section.

**Comment by andydotxyz** [comment]
Hi @elibroftw I watched your video and noticed the troubles you encountered.
Unfortunately many of them were due to accidental characters on your terminal, or the paste not working on your computer (not sure about that - maybe an MSYS2 bug?).

What I also noticed was that you needed to hack around with some of the commands we have in the instructions due to being in the wrong directory, or our hints about the helper commands not being formatted obviously.
I have updated the windows documentation so hopefully it is clearer for people coming along later.
https://developer.fyne.io/started/
